### PR TITLE
Review third party integration

### PR DESCRIPTION
**Pull request by timbussmann**
relates to https://github.com/Particular/docs.particular.net/issues/2174

### CODE REVIEW

File: nservicebus/messaging/third-party-integration.md [hunk 1]
```diff
@@ -1,43 +1,37 @@
...
-- ServicePulse
 related:
 - nservicebus/messaging/headers
+reviewed: 2017-01-18
 ---
 
-When an NServiceBus endpoint receives messages from external systems (such as BizTalk, TIBCO, etc) the message itself might not contain regular all information that NServiceBus provides. That information is necessary for proper serialization of the message, as well as better monitoring and troubleshooting experience.
-
-To get the best use of the tooling ensure that the following information is included in the message sent by the third party endpoints.
+NServiceBus endpints can receive messages from external systems (such as BizTalk, TIBCO, etc). To ensure those messages can be handled correctly by NServiceBus, additional information might be required which are otherwise provided by NServiceBus automatically.
```
Comment (SimonCropp) [inline nservicebus/messaging/third-party-integration.md]: endpints ;)

Comment (indualagarsamy) [inline nservicebus/messaging/third-party-integration.md]: Also add another sentence to further state that, when receiving messages from these external systems, it is therefore important to ensure that the user adds this extra information. The next question then becomes how?

Comment (timbussmann) [inline nservicebus/messaging/third-party-integration.md]: > Also add another sentence to further state that, when receiving messages from these external systems, it is therefore important to ensure that the user adds this extra information.

does this really need to be spelled out besides what's stated on the next sections?

> The next question then becomes how?

Seems to make sense to give more details to that, but that seems to be very transport specific?

**Comment by timbussmann** [comment]
ping @dbelcham

**Comment by indualagarsamy** [comment]
@dbelcham - any updates?

**Comment by indualagarsamy** [comment]
@timbussmann @dbelcham - any updates on this PR?

**Comment by indualagarsamy** [comment]
@timbussmann  - can you fix the build error?

**Comment by timbussmann** [comment]
it didn't look like the build error was related to this PR. Did a rebase and checking again for the build server.

**Comment by timbussmann** [comment]
build seems to be good now @indualagarsamy

**Comment by indualagarsamy** [comment]
@timbussmann @dbelcham - Can either of you look at the task list in the original issue and check off the items, if you've reviewed them and then I'll be happy to merge this. 

 Clean up grammar and wording
 Verify headers listed are complete
 Include header option + serialization options in Required Information
 Make version dependent page
 eliminate reference to "NServiceBus 4 or lower"

**Comment by timbussmann** [comment]
@indualagarsamy checked off the tasks I believe I addressed with this pr. @dbelcham and I discussed to find another solution on how to better integrate serializer specific options (somehow move that to the serializer docs) but that will be a different PR. Happy to keep the related issue and assigned.

**Pull request merged**